extension-content.js

```diff
@@ -1,11 +1,23 @@
+let Keyboard = keyboardEnum();
+
 let filesFilterValue = '';
 let filesFilterInputGlobal;
+let currentCommentPage = -1;
+
 
 document.body.addEventListener('keydown', (event) => {
-  if (event.altKey === true && event.keyCode === 84) {
+  if (event.altKey === true && event.keyCode === Keyboard.KEY_T) {
```

Comment:
Lets use the `code` property of the `event` instead of using the `extension-keyboard-enum.js` file. 
Other than that, it looks nice, thanks for implementing this feature :)

Reply:
By `code` property you mean `KeyboardEvent.code` method, correct?
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code

Also, let me know when it's ok to merge this, so I can rebase it before that.